app/src/main/java/org/sopt/at/views/signin/SignInActivity.kt

```diff
@@ -0,0 +1,447 @@
+                    }
+
+                    append("과 ")
+
+                    withStyle(style = SpanStyle(textDecoration = TextDecoration.Underline)) {
+                        append("서비스 약관")
+                    }
+
+                    append("이 적용됩니다.")
+                },
+                fontSize = 14.sp,
+                color = Color.Gray,
+                modifier = Modifier.align(Alignment.CenterHorizontally).alpha(0.5f),
+                textAlign = TextAlign.Center
+            )
+        }
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun SignInEditText(
+    value: String,
+    type: LoginFieldType,
+    onTextChange: (String) -> Unit,
+    onIconClick: () -> Unit,
+    focusRequester: FocusRequester = FocusRequester(),
+    nextFocusRequester: FocusRequester? = null
+) {
+    val focusManager = LocalFocusManager.current
+    val interactionSource = remember { MutableInteractionSource() }
+    var pwIconState by remember { mutableStateOf(false) }
+
+    BasicTextField(
+        value = value,
+        onValueChange = { onTextChange(it) },
+        singleLine = true,
+        maxLines = 1,
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(horizontal = 16.dp)
+            .focusRequester(focusRequester),
+        textStyle = TextStyle(
+            color = MaterialTheme.colorScheme.onSurface,
+            fontSize = 16.sp
+        ),
+        keyboardOptions = KeyboardOptions(
+            imeAction = if (type == LoginFieldType.EMAIL) ImeAction.Next else ImeAction.Done,
+            keyboardType = if (type == LoginFieldType.EMAIL) KeyboardType.Email else KeyboardType.Password
+        ),
+        keyboardActions = KeyboardActions(
+            onNext = {
+                nextFocusRequester?.requestFocus()
+            },
+            onDone = {
+                focusManager.clearFocus()
+            }
+        ),
+        visualTransformation = when {
+            type == LoginFieldType.PASSWORD && !pwIconState -> PasswordVisualTransformation()
+            else -> VisualTransformation.None
+        },
+        decorationBox = { innerTextField ->
+            TextFieldDefaults.DecorationBox(
+                value = value,
+                innerTextField = innerTextField,
+                enabled = true,
+                singleLine = true,
+                interactionSource = interactionSource,
+                isError = false,
+                label = null,
+                placeholder = {
+                    Text(
+                        text = if (type == LoginFieldType.EMAIL) "아이디" else "비밀번호",
+                        color = MaterialTheme.colorScheme.onSurface.copy(alpha = 0.5f),
+                        fontSize = 14.sp
+                    )
+                },
+                trailingIcon = {
+                    if (type == LoginFieldType.EMAIL && value.isNotEmpty()) {
+                        IconButton(onClick = { onIconClick() }) {
```

Comment:
인자가 없는 경우, 혹은 함수의 인자 순서와 람다의 변수 순서가 같다면 람다 없이 함수이름만 적으셔도 됩니다!
```suggestion
                        IconButton(onClick = onIconClick)
```

Reply:
명확하게 람다 호출을 하고 있다는 것과 확장을 생각해서 보통 저렇게 많이 사용하는 편인데 간결하게 작성하는 방식 역시 보기 좋네요! 나중에 적용 해보겠습니다!